lang/ar.js

```diff
@@ -7,11 +7,11 @@ export default {
 	"components.calendar.selected": "تم التحديد.",
 	"components.calendar.show": "إظهار {month}",
 	"components.dialog.close": "إغلاق مربع الحوار هذا",
-	"components.filter.loading": "Loading filters",
-	"components.filter.filterCount": "{number, select, max {99+} all {كل} other {{من الأرقام}}}",
+	"components.filter.loading": "",
```

Comment:
I think most of these ones that are just English got added after the translations were sent out -- so I'd expect them to arrive in the next batch. Is it better to leave the English in here vs. having them be empty?

Reply:
I personally prefer to leave it empty since that makes it obvious what's missing and the prepare translations action will fill them in withe english anyways.

Reply:
Ah gotcha 👍